apps/01PigLatin.js

```diff
@@ -3,49 +3,92 @@
 var assert = require('assert');
 var readline = require('readline');
 var rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
+    input: process.stdin,
+    output: process.stdout
 });
 
 
 function pigLatin(word) {
 
-  // Your code here
+    //  var word = 'car';
+    var firstLetter = word[0];
+    console.log(firstLetter);
+    word = word.toLowerCase();
 
+    var word = 'crazy';
+    var vowelIndex = -1; // Set it to assume there are no vowels
+
+    if ((word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex) || vowelIndex === -1) {
+        vowelIndex = word.indexOf('a');
+    }
+
+    if ((word.indexOf('e') > -1 && word.indexOf('e') < vowelIndex) || vowelIndex === -1) {
+        vowelIndex = word.indexOf('e');
+    }
+
+    if ((word.indexOf('i') > -1 && word.indexOf('i') < vowelIndex) || vowelIndex === -1) {
+        vowelIndex = word.indexOf('i');
+      }
+    if ((word.indexOf('o') > -1 && word.indexOf('o') < vowelIndex) || vowelIndex === -1) {
+          vowelIndex = word.indexOf('o');
+      }
+
+      if ((word.indexOf('u') > -1 && word.indexOf('u') < vowelIndex) || vowelIndex === -1) {
+          vowelIndex = word.indexOf('u');
+      }
+
+      if ((word.indexOf('y') > -1 && word.indexOf('y') < vowelIndex) || vowelIndex === -1) {
+          vowelIndex = word.indexOf('y');
+      }
```

Comment:
please fix your indentation here:

select lines 36-42
use `command`+`[` to indent them all to the left